static/app/views/nav/secondary/sections/explore/exploreSecondaryNav.tsx

```diff
@@ -1,14 +1,18 @@
+import {useRef} from 'react';
+
 import Feature from 'sentry/components/acl/feature';
 import {t} from 'sentry/locale';
 import useOrganization from 'sentry/utils/useOrganization';
 import {useGetSavedQueries} from 'sentry/views/explore/hooks/useGetSavedQueries';
-import {getExploreUrlFromSavedQueryUrl} from 'sentry/views/explore/utils';
 import {PRIMARY_NAV_GROUP_CONFIG} from 'sentry/views/nav/primary/config';
 import {SecondaryNav} from 'sentry/views/nav/secondary/secondary';
+import {ExploreSavedQueryNavItems} from 'sentry/views/nav/secondary/sections/explore/exploreSavedQueryNavItems';
 import {PrimaryNavGroup} from 'sentry/views/nav/types';
 
 export function ExploreSecondaryNav() {
   const organization = useOrganization();
+
+  const sectionRef = useRef<HTMLDivElement>(null);
```

Comment:
Why does this need to be defined up here? It doesn't seem like it's attached to any element in this file nor is there any logic attached to it. Maybe we can push it down?